### PR TITLE
Move pipeline-service-tests logic to shell scripts

### PR DESCRIPTION
**Pull request by Roming22**
This change aims to make the pipeline yaml more readable by abstracting the logic in scripts, as well as being able to apply the CI to the content of those scripts.

Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

File: .tekton/pipeline-service-test.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ spec:
       - name: shared-workspace
     tasks:
       - name: produce-cluster-name
+        runAfter:
+          - "fetch-repository"
```
Comment (bnallapeta) [inline .tekton/pipeline-service-test.yaml:35]: Is there a reason this has to be run after fetch-repository?

File: .tekton/pipeline-service-test.yaml [hunk 2]
```diff
@@ -48,61 +50,25 @@ spec:
                 echo -n "$CLUSTER_NAME" | tee $(results.cluster-name.path)
       - name: clean-clusters
         runAfter:
-          - "produce-cluster-name"
-        params:
-          - name: namespace
-            value: "$(params.namespace)"
-          - name: cluster-name
-            value: "$(tasks.produce-cluster-name.results.cluster-name)"
+          - "fetch-repository"
```
Comment (bnallapeta) [inline .tekton/pipeline-service-test.yaml:53]: Shouldn't this ideally run after everything is done as the last step? And not after fetch-repository?

**Review by bnallapeta — APPROVED** [review]
LGTM apart from those minor comments.
@xinredhat PTAL and merge if it looks good to you.

File: .tekton/pipeline-service-test.yaml (hunk 1 above)
Comment (Roming22) [reply .tekton/pipeline-service-test.yaml:35]: It makes a bit more sense when looking at flow in the ui, as `delete-clusters` does not depend on that task.

**Pull request merged**